lib/workers/repository/process/fingerprint-fields.ts

```diff
@@ -0,0 +1,187 @@
+export const actualBranchConfigFields = [
```

Comment:
This is unmaintainable, we should not continue down this approach. Please stop work and go back to the issue to discuss approaches